### PR TITLE
Add parameters plot functions

### LINKED ISSUE #398: Add parameters to 'plot_*' functions
Currently, there is no way to control the parameters of `prep_*` functions used inside of the `plot_*` functions for techmix and emission intensity. This feature should be added to them as it was done for trajectory (see #395 ).

### PR DESCRIPTION
**Pull request by MonikaFu**
Closes #398 

In this PR I expose the parameters of `prep_*()` functions in the `plot_*()` functions. Following the change that was already introduced for trajectory charts in #395.

### CODE REVIEW

File: tests/testthat/test-plot_emission_intensity.R [hunk 1]
```diff
@@ -45,3 +45,31 @@ test_that("with too many lines to plot errors gracefully", {
...
+  expect_false(isTRUE(all.equal(labels_def, labels_mod)))
+})
+
+test_that("is sensitive to `span_5yr`", {
+  data <- filter(sda, sector == "cement")
+
+  p_f <- plot_emission_intensity(data, span_5yr = FALSE)
+  min_year <- min(as.numeric(format(p_f$data$year, format = "%Y")), na.rm = TRUE)
+  max_year <- max(as.numeric(format(p_f$data$year, format = "%Y")), na.rm = TRUE)
+  expect_false(max_year - min_year == 5)
```
[the remark this answers is not in the document]
Comment (MonikaFu) [reply tests/testthat/test-plot_emission_intensity.R:69]: In general yes but assuming that the test dataset is fixed - no. But if this is not the right practice and we should make the year difference more explicit in the test data - we can do it.

File: R/plot_emission_intensity.R [hunk 2]
```diff
@@ -14,12 +21,19 @@
 #' # `data` must meet documented "Requirements"
 #' data <- subset(sda, sector == "cement")
 #' plot_emission_intensity(data)
-plot_emission_intensity <- function(data) {
+plot_emission_intensity <- function(
+  data,
+  convert_label = identity,
+  span_5yr = FALSE
+  ) {
```
[the remark this answers is not in the document]
Comment (maurolepore) [reply R/plot_emission_intensity.R]: /style

File: tests/testthat/test-plot_emission_intensity.R (hunk 1 above)
[the remark this answers is not in the document]
Comment (maurolepore) [reply tests/testthat/test-plot_emission_intensity.R]: I pushed a PR with an assertion: https://github.com/2DegreesInvesting/r2dii.plot/pull/424

File: R/plot_emission_intensity.R [hunk 3]
```diff
@@ -14,12 +21,19 @@
 #' # `data` must meet documented "Requirements"
 #' data <- subset(sda, sector == "cement")
 #' plot_emission_intensity(data)
-plot_emission_intensity <- function(data) {
+plot_emission_intensity <- function(
```
[the remark this answers is not in the document]
Comment (MonikaFu) [reply R/plot_emission_intensity.R]: See a59610d and let me know if this is what you meant here.

**Pull request merged**